content/updates/2017-11-20-release-notes-buildpacks-volume-services-other-new-features.md

```diff
@@ -0,0 +1,54 @@
+* We have experimental support for [volume services](https://cloud.gov/docs/apps/experimental/volume-services/). Since a cloud.gov app normally has a short-lived file system, this enables you to build an app that requires persistent file storage. [Contact us if you’d like to try this](https://cloud.gov/docs/apps/experimental/volume-services/#how-to-use-this-service).
+* One of our engineers gave us the ability to make [Mermaid diagrams](https://mermaidjs.github.io/) on the cloud.gov website, and we almost got carried away creating visualizations of different parts of the platform. Here are a couple we think will help you out:
+  * A comparison of how apps with custom domains and those with the default \*.app.cloud.gov domain are served to users: https://cloud.gov/docs/apps/custom-domains/#comparison-of-default-domains-and-custom-domains.
+  * How to incorporate a CI/CD workflow into your app: https://cloud.gov/docs/apps/continuous-deployment/#configure-your-service
+* The cloud.gov dashboard now lets you manage environment variables for your apps. To see it in action, go to an app page in the dashboard and try viewing, adding, or editing an environment variable. You can also see all the environment variables set by the system from the same view.
+
+<img src="/img/dashboard-envs.png" alt="Screenshot of the environment variable editing view on the dashboard" border=5 />
+
+### Updates
+
+Make sure to use the latest version of the Cloud Foundry CLI. The most recent updates contain new commands and improvements to existing commands that will make your cloud.gov workflow faster and easier. You can download the latest binary for Windows, Mac, or Linux from GitHub: https://github.com/cloudfoundry/cli/releases
+
+### Announcement
+
+We will not, as we previously announced, be switching users with cloud.gov accounts to sign in with MAX.gov.
+
+### Coming soon
+
+#### Sandboxes will expire after 90 days
+
+We plan to automatically deletes apps in sandbox spaces after 90 days. We’ll send email notifications before this happens. If you’re developing something you need to keep long-term, we recommend moving it to a prototyping space. If you’d like to start the process of purchasing a prototyping package, get in touch with us.
+
+#### Additional TIC compliance support
+
+To support agency implementations of Trusted Internet Connection (TIC) requirements, we’ve published documentation about complying with TIC for apps on cloud.gov. To support a wider range of agency TIC needs, we’ve also built a way to support restricting users to trusted IP ranges. We’re waiting to fully implement it until after it’s passed FedRAMP review, but in the meantime, let us know if you have questions or want to use it.
```

Comment:
Suggest linking to https://cloud.gov/docs/compliance/meeting-tic-requirements/ and https://cloud.gov/docs/apps/experimental/restricting-users-to-trusted-ip-ranges/